tests/unit/calendar/calendar_test_helpers.js

```diff
@@ -16,9 +16,9 @@ TestHelpers.calendar = {
 	},
 	focusGrid: function( element ) {
 		element.find( ":tabbable" ).last().simulate( "focus" );
```

Comment:
Should store the selected element and use it for the next three calls, including the return.